pkg/runctl/run_manager.go

```diff
@@ -568,6 +568,33 @@ func (c *runManager) createTektonTaskRun(ctx *runContext) error {
 	return err
 }
 
+func (c *runManager) addTektonTaskRunParamsForImage(
+	pipelineRun k8s.PipelineRun,
+	tektonTaskRun *tekton.TaskRun,
+) {
+	spec := pipelineRun.GetSpec()
+	jfrSpec := spec.JenkinsfileRunner
+	image := c.pipelineRunsConfig.Image
+	imagePullPolicy := c.pipelineRunsConfig.ImagePullPolicy
+
+	if jfrSpec != nil {
+		if jfrSpec.Image != "" {
+			image = jfrSpec.Image
+			if jfrSpec.ImagePullPolicy == "" {
```

Comment:
Also check if `imagePullPolicy` is already set. We should not override it in this case.

Reply:
This is done by intention. See https://github.com/SAP/stewardci-core/pull/162#discussion_r503222042